### PR TITLE
Rework implementation to use VideoMaster C++ Wrapper

### PR DESCRIPTION
**Pull request by laurent-radoux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +32,7 @@ After installing the SDK according to the official documentation, the libs and h
 
 If you used Conan to retrieve your dependencies, you can use the following commands to build the project:
 
-    cmake -S . -B build -DCMAKE_MODULE_PATH:PATH=/path/to/modules
+    cmake -S . -B build -DCMAKE_TOOLCHAIN_FILE=generators/conan_toolchain.cmake
```
Comment (Nathoufresh) [inline README.md:35]: This command is not working as is on Windows. Further explanations might be required.

File: src/helper.cpp [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        os << "\t" << "\t" << "- SCP: 0x" << board.scp().version() << std::endl;
+    os << std::dec;
+
+    return os;
+}
+
+namespace Application::Helper
+{
+    std::optional<std::reference_wrapper<BoardComponents::Loopback>> get_loopback(Board& board, unsigned int channel_index)
+    {
```
Comment (Nathoufresh) [inline src/helper.cpp:49]: I don't see were we change the loopback state. Should we keep it?

File: README.md [hunk 3]
```diff
@@ -15,16 +15,12 @@ See https://www.deltacast.tv for more video products.
 VideoViewer requires some dependencies to be installed on the system:
```
Comment (Nathoufresh) [inline README.md:15]: Maybe we should also state that this repo uses git submodules. `git clone ... --recurse-submodules `

**Review by Nathoufresh — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (laurent-radoux) [reply README.md:15]: Done

File: src/helper.cpp (hunk 2 above)
Comment (laurent-radoux) [reply src/helper.cpp:49]: I forgot to add the enable/disable of loopback in the main. Just done it.

File: README.md (hunk 1 above)
Comment (laurent-radoux) [reply README.md]: Switched to `--preset`

**Review by Nathoufresh — APPROVED** [review]

**Pull request merged**